src/router.rs

```diff
@@ -108,6 +109,35 @@ impl RouteInfo {
         }
         false
     }
+
+    /// Check if application contains matching route.
+    ///
+    /// This method does take `prefix` into account.
+    /// For example if prefix is `/test` and router contains route `/name`,
+    /// following path would be recognizable `/test/name` and `has_route_prefixed()` call
+    /// would return `true`.
+    pub fn has_route_prefixed(&self, path: &str) -> bool {
```

Comment:
Suggest for name: `has_route_with_prefix`

Reply:
That's what I named it initially, but that would semantically mean that it would accept a second argument (ie: `prefix: &str`). I can still use `has_route_with_prefix` if you'd like